CMakeLists.txt

```diff
@@ -216,7 +216,9 @@ option(QUDA_TEX "enable texture reads?" ON)
 option(QUDA_FLOAT8 "enable float-8 ordered fixed-point fields?" OFF)
 option(QUDA_NVML "use NVML to report CUDA graphics driver version" OFF)
 option(QUDA_NUMA_NVML "experimental use of NVML to set numa affinity" OFF)
-option(QUDA_VERBOSE_BUILD "display kernel register useage" OFF)
+option(QUDA_VERBOSE_BUILD "display kernel register usage" OFF)
+option(QUDA_CUDA_LAPACK "use cuBLAS lapack" ON)
```

Comment:
Just wondering if the option should be `QUDA_BLAS_LAPACK` with the options being `NATIVE` or `GENERIC`.  On CUDA, `NATIVE` would correspond to cuBLAS.  This would allow one to use the generic variant on CUDA if desired, e.g., for benchmarking.